neurodamus/ngv.py

```diff
@@ -135,15 +137,17 @@ def _init_cell(gid, morph_file):
 
         # Insert mechanisms and populate holder lists
         logging.debug("Instantiating NGV cell gid=%d", gid)
+        print("Instantiating NGV cell gid=%d" % (gid))
```

Comment:
Not sure that we need to print this msg again. There is also ready the debug log for the same content. If you run neurodamus in debug mode, that will be printed.